app/dfda/components/StudyCard.tsx

```
@@ -0,0 +1,168 @@
+          <h2 className="neobrutalist-h2 mb-6">Study Data</h2>
+          <div className="grid grid-cols-1 gap-8">
+            {study.studyCharts.correlationScatterPlot && (
+              <HighchartsReact
+                highcharts={Highcharts}
+                options={study.studyCharts.correlationScatterPlot.highchartConfig}
+              />
+            )}
+            {study.studyCharts.pairsOverTimeLineChart && (
+              <div className="neobrutalist-container bg-gray-50 p-4">
+                <h3 className="neobrutalist-h3 mb-4">Data Over Time</h3>
+                <div dangerouslySetInnerHTML={{ 
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)